pyproject.toml

```diff
@@ -28,8 +28,7 @@ dependencies = [
     "packaging >= 20.0",
     # pandas -> For data loading
     "pandas >=1.0.0,<2.0.0",
-    # Aligned pydantic version with server fastAPI
-    "pydantic >= 1.10.7,< 2.0",
+    "pydantic >=2.0.0",
```

Comment:
We should also support users using pydantic < 2.0. So, these changes won't be compatible with pydantic 1.x users, right?

Reply:
yes, that's true. With my latest change `argilla` should be compatible with both versions.